### PR TITLE
fix testing and include students feedback

### PR DESCRIPTION
**Pull request by anilder89**
Documentation content update for [Module IBC, Section checkers and leaderboard]

This PR includes adjustments for the tests and some highlight boxes to address feedback. 
### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [X] Small content fixes 
* [X] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

File: hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md [hunk 1]
```diff
@@ -77,6 +77,59 @@ message Board {
...
+and add leaderboard's keeper into it:
+
+```golang
+    leaderboardKeeper,_ := LeaderboardKeeper(t);
+    k := keeper.NewKeeper(
+        bank,
+        *leaderboardKeeper,
+        cdc,
+        storeKey,
+        memStoreKey,
```
Comment (xavierlepretre) [inline hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md:130]: I would adjust the indentation on all of them. 4 spaces, No `TAB` and always start at the same level a the backticks.

**Review by AndrewB9lab — APPROVED** [review]
Language review 01 completed.

File: hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ message Board {
 Note that you will also have to modify the `x/leaderboard/genesis.go`. In it, look for:
 
 ```golang
-// Set if defined
+    // Set if defined
     if genState.Board != nil {
         k.SetBoard(ctx, *genState.Board)
     }
```
Comment (xavierlepretre) [inline hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md:88]:
```suggestion
// Set if defined
if genState.Board != nil {
    k.SetBoard(ctx, *genState.Board)
}
```
Why not like this?

Comment (anilder89) [reply hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md:88]: Just to indicate that this piece of code belongs to some function.

**Review by coldice — APPROVED** [review]

**Pull request merged**